dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -132,9 +134,10 @@ DataInterfaceIIDM::build(std::string iidmFilePath) {
     if (getEnvVar("DYNAWO_USE_XSD_VALIDATION") == "true")
       xsdValidation = true;
 
-    IIDM::Network networkIIDM = parser.from_xml(iidmFilePath, xsdValidation);
+    boost::shared_ptr<IIDM::Network> networkIIDM = boost::make_shared<IIDM::Network>(parser.from_xml(iidmFilePath, xsdValidation));
     data.reset(new DataInterfaceIIDM(networkIIDM));
     data->initFromIIDM();
+    data->importStaticParameters();
```

Comment:
Why do we need to call this here?

Reply:
Will be removed. Artefact of previous factorization